resources/views/community/components/claims/finished-claim-table-row.blade.php

```diff
@@ -5,7 +5,9 @@
 
 $claimSetTypeCopy = ClaimSetType::toString(ClaimSetType::NewSet);
 if ($claim['SetType'] !== ClaimSetType::NewSet) {
-    $claimSetTypeCopy = ClaimSetType::toString(ClaimSetType::Revision);
+    $claimSetTypeCopy = '🗒️ ' . ClaimSetType::toString(ClaimSetType::Revision);
+} else {
+    $claimSetTypeCopy = "💡 <b>$claimSetTypeCopy</b>";
```

Comment:
```suggestion
    $claimSetTypeCopy = "💡 <span class='font-bold'>$claimSetTypeCopy</span>";
```

`<span>` instead of `<b>` so we are at the mercy of Tailwind's defaults, not the browser's. It is also slightly better for accessibility.

But I'm not sure we should be highlighting new sets in this way. There are some incredible revisions being released nowadays, and bolding New but not bolding Revision may send the wrong message.